packages/quick_actions/quick_actions_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
```

Comment:
@stuartmorgan should this be NEXT since it only impacts the example app?

Reply:
Yes, only the published file (example/lib/main.dart for all of our plugins) should be flagged by the CI as needing a version change.

It's also fine to just fully remove this and override the changelog change for this (see https://github.com/flutter/flutter/issues/111560).

Reply:
Friendly ping @hellohuanlin